src/confirmation.rs

```diff
@@ -0,0 +1,79 @@
+use crate::zulip::api::ZulipApi;
+use crate::{get_env, run_diffs, ServiceDiff};
+use sha2::{Digest, Sha256};
+
+pub(crate) struct Confirmation {
```

Comment:
Nit: let's leave a comment here to explain the confirmation is of. I think a one sentence explanation should be enough.